### DISCUSSION TITLE
action stderr is logging debug messages

### DISCUSSION THREAD

**Discussion by guzzijones** [StackStorm/st2#5465] [Q&A] [answered]
I have my logging set to INFO, but I am still seeing debug messages in the stderr output of an action.  Where do you set the log level for an action's stderr?

**Comment by guzzijones** [comment, marked as answer]
Oh there is a `log_level` parameter on the action parameter for python actions:
https://github.com/StackStorm/st2/blob/5fa48d11801a3415b7122db3058efadefafbf815/conf/st2.conf.sample#L20

**Comment by akshat-rubrik** [comment]
But why is it in stderr?
